Content/default/Build.fs

```diff
@@ -1,20 +1,18 @@
-#r "paket: groupref build //"
-#load "./.fake/build.fsx/intellisense.fsx"
-#r "netstandard"
-
 open Fake.Core
 open Fake.DotNet
 open Fake.IO
 open Farmer
 open Farmer.Builders
 
-Target.initEnvironment ()
+let execContext = Context.FakeExecutionContext.Create false "build.fsx" [ ]
+Context.setExecutionContext (Context.RuntimeContext.Fake execContext)
```

Comment:
I think we should add a FAKE function to do this so we can just write something like `Target.initExeEnvironment ()`. Or maybe even remove the need for this if possible when using FAKE this way.

Reply:
Yup it is a boilerplate, but I could live with those two extra lines